### PR TITLE
[Merged by Bors] - TY-2658 replace real domains

### PR DESCRIPTION
**Pull request by janpetschexain**
**References**

- [TY-2658]

**Summary**

- replace real domains with example domains

[TY-2658]: https://xainag.atlassian.net/browse/TY-2658?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: discovery_engine/test/feed_manager_test.dart [hunk 1]
```diff
@@ -247,25 +247,26 @@ Future<void> main() async {
...
 
       final excludedSoures = {
-        Source('theguardian.com'),
-        Source('bbc.co.uk'),
-        Source('wsj.com'),
-        Source('www.nytimes.com'),
+        Source('example.com'),
+        Source('example.org'),
+        Source('example.net'),
+        Source('www.example.info'),
```
Comment (acrrd) [inline discovery_engine/test/feed_manager_test.dart:269]: :wrench: This is a valid domain. a part from `example.com`, `example.org`, `example.net`, other domain must in the form `*.example`, unless a standard is saying that they should be used only for documentation/testing pupose.

File: discovery_engine_core/core/test-fixtures/articles-some-malformed-media-urls.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     ],
     "media": "",
     "is_opinion": false,
-    "twitter_account": "@XaynHQ",
+    "twitter_account": "@Example",
```
Comment (acrrd) [inline discovery_engine_core/core/test-fixtures/articles-some-malformed-media-urls.json:21]: :wrench: This is a valid twitter account. I guess we can use `xaynhq` here. I'm not sure if there is some handle reserved for this use case.

File: discovery_engine/test/feed_manager_test.dart (hunk 1 above)
Comment (janpetschexain) [reply discovery_engine/test/feed_manager_test.dart:269]: good point. i'll also change other occurences like example.de

File: discovery_engine_core/core/test-fixtures/articles-some-malformed-media-urls.json (hunk 2 above)
Comment (janpetschexain) [reply discovery_engine_core/core/test-fixtures/articles-some-malformed-media-urls.json:21]: well that is unexpected :O looks like the twitter api doesn't provide a dummy handle, i'll revert this change.

**Review by acrrd — APPROVED** [review]

**Comment by janpetschexain** [comment]
bors merge

**Pull request closed without merging**